src/actions/nodes.js

```diff
@@ -59,7 +59,7 @@ export const ADD_NODE_METADATA = 'ADD_NODE_METADATA';
 
 /**
  * Toggle whether to display the loading spinner
- * @param {boolean} loading True if pipeline is still loading
+ * @param {Object} data The node's data
```

Comment:
I don't have much context here; feel free to suggest a better description, if necessary.